src/udunits2_R.c

```diff
@@ -40,10 +42,20 @@ void handle_error(const char *calling_function) {
   error("Error in function %s: %s", calling_function, ut_status_strings[stat]);
 }
 
+void R_ut_reinit(const char **path) { /* try to reboot the init with a new path to the database */
+	if (strlen(*path) > 255)
+  		error("path too long");
```

Comment:
I've run into problems before (particularly with running and old Python packaging software, pip, on an ecyrptfs), where the software wanted to create a long path, the fs didn't support it, and as a user, I was just SOL. Could we make the error message here to be something that tells the user what action should be taken? Even if that's just "file a bug against Rudunits2 and ask for a larger buffer". That at least gives the user some indication of what can be done to alleviate the problem.